docs/source/conf.py

```diff
@@ -22,7 +22,7 @@
 author = "Matteo Lodi"
 
 # The full version, including alpha/beta/rc tags
-release = "1.9.0"
+release = "1.9.1"
```

Comment:
same, shouldn't be part of this PR. I think there is need to pull and merge once.